tests/sidebar-ol.test.js

```diff
@@ -0,0 +1,128 @@
+// work around jsdom canvas issue in jest
+HTMLCanvasElement.prototype.getContext = jest.fn()
+
+var ol = require('openlayers');
+var Sidebar = require('../js/ol3-sidebar');
+
+describe('Simple sidebar object instantiation', function() {
+  beforeEach(function() {
+    document.body.innerHTML =
+      '<div id="sidebar" class="sidebar collapsed">' +
+      '  <div class="sidebar-tabs">' +
+      '  </div>' +
+      '  <div class="sidebar-content">' +
+      '  </div>' +
+      '</div>' +
+      '<div id="map" class="sidebar-map"></div>';
+  });
+
+  test('Sidebar object minimal instantiation', function() {
+    var sidebar = new ol.control.Sidebar({ element: 'sidebar' });
+
+    expect(sidebar._options.position).toBe('left');
+    expect(sidebar._options.element).toBe('sidebar');
+    expect(sidebar._tabitems.length).toBe(0);
+    expect(sidebar._panes.length).toBe(0);
+    expect(sidebar._closeButtons.length).toBe(0);
```

Comment:
these are testing implementation details of the `Sidebar` class. we should be testing the resulting HTML instead.

Reply:
I know, and this was something I wanted to get rid of over time.  It wasn't very clear what a "finished" bare-bones Sidebar object actually looked like and in order to test this I had to dig into the internals (which I know is a dodgy thing, but it helped get this far). However you're right, these tests are effectively covered by later, more involved tests and hence can be removed.

Reply:
I've replaced these `expect` statements with a simply truthy check.  Hope that's ok :-)